programs/lfx-mentorship/README.md

```diff
@@ -1,10 +1,10 @@
-# LFX Mentorship (ex-Community Bridge) by The Linux Foundation
+# LFX Mentorship by The Linux Foundation
```

Comment:
Can we make this change on main first?

Personally, I would want to see this PR only doing these:
- Moving files
- Changing links to new file locations
- Adding new files
- Deleting obsolete files

But, IMO, we should not change the content here on this PR.

Commit that changed this content is: https://github.com/cncf/mentoring/commit/b4f5efdc

Reply:
I agree.

Reply:
Looking at it -- it's within a commit that moves a bunch of other content. In order to pull this out, we'll need to create a commit removing it, then create another commit to re-add it after. I'm not sure it's worth the effort to remove it given we'll need to immediately re-add it.

Reply:
I see... The commit has more than just the change I highlighted above, so, can't do a simple cherrypick. I am ok with leaving it as is :)